### ISSUE TITLE
✨ dagger uninstall

### ISSUE THREAD

**Issue by mircubed** [dagger/dagger#5559]
### What are you trying to do?

we should have a command to easily uninstall when needed 

### Why is this important to you?

An easy uninstall command makes it easier for people to try Dagger without getting frustrated when and if they ever need to remove it. 

### How are you currently working around this?

We are building docs for the current uninstall process, but currently users have to figure out how to do it on their own.

**Comment by mircubed** [comment]
We should make a note to update the "uninstall" documentation when this is completed since the documentation for the current workaround in #2123 will be documented before this is done.

**Comment by TomChv** [comment]
@vikram-dagger Any thought on that one?

**Comment by vikram-dagger** [comment]
We have https://docs.dagger.io/faq/#how-do-i-uninstall-dagger

**Comment by gerhard** [comment]
Yes, we should definitely add this. Just confirmed with @helderco.
